modules/core/output_modules.php

```diff
@@ -314,7 +314,7 @@ class Hm_Output_server_content_start extends Hm_Output_Module {
      */
     protected function output() {
         return '<div class="content_title">'.$this->trans('Servers').
-            '<div class="list_controls"></div>'.
+            // '<div class="list_controls"></div>'.
```

Comment:
Is this comment a temporary thing? If permanent, maybe you just need to remove that line of code.